src/Sylius/Bundle/ApiBundle/Resources/config/services.xml

```diff
@@ -15,36 +15,45 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
 >
-<!--    <imports>-->
-<!--        <import resource="services/**/*.xml" />-->
```

Comment:
I think we should leave this line commented as we should go back to importing everything in one line to not forget about it